edge/services/cpu_percent/Dockerfile.amd64

```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk --no-cache --update add gawk bc socat
+RUN apk --no-cache --update add gawk bc socat curl
```

Comment:
Why did you need to add curl to the cpu_percent and gps docker images? The new testing method uses curl on the host, so doesn't need it in the container, right? Or am i missing something?

Reply:
After some experimentation it appears that they do require curl for this to work. I believe it's because I am exec-ing into the container and running curl, not curling from the outside since they don't "expose" their ip address, or whatever that docker flag is to expose their info.

Reply:
makes sense